ocaml/xapi/quicktest_bvt.ml

```diff
@@ -0,0 +1,40 @@
+open Client
+open Quicktest_common
+
+(* alias PBD module for convenience 
+ * also has scope for generalising verification
+ * between PBDs and VBDs for common API calls
+ * e.g. both have get_uuid, get_other_config etc. *)
+module P = Client.PBD
+
+let start session_id rpc =
+
+  (* helper functions for API/PBD calls *)
+  let with_api fn = fn ~rpc ~session_id in
+  let with_pbd fn pbd = with_api fn ~self:pbd in
+
+  (* checks get_uuid returns correct UUID by comparing
+   * the PBD returned by get_by_uuid *)
+  let uuid_test pbd =
+    let uuid_t = make_test "Testing 'get_by_uuid', 'get_uuid'" 4 in
+    try
+      start uuid_t;
+      let uuid = with_pbd P.get_uuid pbd in
+      if with_api P.get_by_uuid ~uuid = pbd then success uuid_t
```

Comment:
The indentation of `if..then..else` is not following the logical structure. Please use a more conventional way.